parl/utils/exceptions.py

```diff
@@ -40,6 +43,10 @@ class DeserializeError(UtilsError):
     """
 
     def __init__(self, error_info):
+        error_info = (
+            'Deserialize error, you may have provided a pyarrow' +
+            'non-deserializable object. Detailed error:\n{}'.format(error_info)
```

Comment:
you may have provided an object that cannot be deserialized by pyarrow.

Reply:
done